src/systemctl/systemctl.c

```diff
@@ -4262,13 +4315,18 @@ static void print_status_info(
         }
 
         ss = streq_ptr(i->active_state, i->sub_state) ? NULL : i->sub_state;
-        if (ss)
-                printf("     Active: %s%s (%s)%s",
-                       active_on, strna(i->active_state), ss, active_off);
-        else
+        if (ss) {
+                if (!streq(i->freezer_state, "running"))
+                        printf("   Active: %s%s (%s)%s",
+                               active_on, strna(i->active_state), i->freezer_state, active_off);
```

Comment:
this hides the substate if we have a freezer state. i think we should show all three states in this case